Cargo.toml

```diff
@@ -104,7 +104,8 @@ slog-term = "2.4"
 slog_derive = "0.1"
 spin = "0.5"
 sst_importer = { path = "components/sst_importer" }
-sysinfo = { git = "https://github.com/tikv/sysinfo.git", branch = "tikv"}
+sysinfo = { git = "https://github.com/nrc/sysinfo.git", branch = "vendor" }
```

Comment:
Would you mind send your changes to tikv/sysinfo?

Reply:
I have (https://github.com/tikv/sysinfo/pull/1), I'm waiting for that to be merged, then I will update the PR